### PR TITLE
Check for existing repo before forking

### PR DESCRIPTION
**Pull request by slide**
Adds a check to see if the repository already existing in the org before trying to fork. 

CC @daniel-beck

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -86,6 +86,21 @@
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.powermock</groupId>
+      <artifactId>powermock-module-junit4</artifactId>
+      <version>1.6.5</version>
```
Comment (daniel-beck) [inline pom.xml]: These should be test scoped.

File: src/main/java/org/jenkinsci/backend/ircbot/IrcBotImpl.java [hunk 2]
```diff
@@ -828,8 +834,6 @@ private boolean forkGitHub(String channel, String sender, String owner, String r
                 return false;
             }
 
-            GHOrganization org = github.getOrganization(IrcBotConfig.GITHUB_ORGANIZATION);
-            GHRepository r;
             try {
                 r = orig.forkTo(org);
```
Comment (daniel-beck) [inline src/main/java/org/jenkinsci/backend/ircbot/IrcBotImpl.java]: I'm not a huge fan of variable reuse.

**Comment by daniel-beck** [comment]
Besides these two minor issues, looks good.

@oleg-nenashev WDYT?

**Comment by oleg-nenashev** [comment]
Looks good to me excepting dependencies

**Comment by slide** [comment]
Updated based on feedback.

**Pull request merged**